target_redshift/db_sync.py

```diff
@@ -218,7 +218,8 @@ def __init__(self, connection_config, stream_schema_message=None):
         self.s3 = boto3.client(
             's3',
             aws_access_key_id=self.connection_config['aws_access_key_id'],
-            aws_secret_access_key=self.connection_config['aws_secret_access_key']
+            aws_secret_access_key=self.connection_config['aws_secret_access_key'],
+            aws_session_token=self.connection_config['aws_session_token'],
```

Comment:
it gives `KeyError: 'aws_session_token'` when `aws_session_token` is not defined in `config.json`

Can we change it to `aws_session_token=self.connection_config.get('aws_session_token')` ?

Reply:
Done